technical-reports/resolver/terminology.md

```diff
@@ -1,103 +1,37 @@
-# Resolver Terminology
+# Terminology
 
-<dl>
-<dt>Resolver</dt>
-<dd>
+## <dfn>Resolver</dfn>
 
-A mechanism that combines token sets and applies modifiers to produce a final, resolved set of design tokens.
+The mechanism by which multiple possible values of design tokens are reduced to a single value, i.e. this module.
 
-</dd>
-<dt>Token Set</dt>
-<dd>
+## <dfn>Set</dfn>
 
-A collection of design tokens grouped together, often representing foundational tokens, component-specific tokens, or theme-specific tokens.
```

Comment:
> A collection of design tokens grouped together, often representing foundational tokens, component-specific tokens, or theme-specific tokens.

This is also an imprecise definition. “A collection of design tokens” is defined as a **group** already according to the spec.

Also, “foundational tokens,” “component-specific tokens,” etc. are **not definitions.** They are **examples!** Not an actual definition.

Improved all of these to try and form some language that communicates that these are _alternate_ design token values, intended to be merged in a specific order.